coreneuron/nrniv/memory.h

```diff
@@ -29,11 +29,58 @@ THE POSSIBILITY OF SUCH DAMAGE.
 #ifndef _H_MEMORY_
 #define _H_MEMORY_
 
-#include <stdlib.h>
 #include <string.h>
+
 #include "coreneuron/nrniv/nrn_assert.h"
 
+#if !defined(NRN_SOA_BYTE_ALIGN)
+// for layout 0, every range variable array must be aligned by at least 16 bytes (the size of the
+// simd memory bus)
+#define NRN_SOA_BYTE_ALIGN (8 * sizeof(double))
+#endif
+
+/// for gpu builds with unified memory support
+#if (defined(__CUDACC__) || (defined(_OPENACC)) && defined(UNIFIED_MEMORY))
```

Comment:
```suggestion
#if (defined(__CUDACC__) || defined(UNIFIED_MEMORY))
```
should be sufficient?

Reply:
Should be sufficient since `UNIFIED_MEMORY` is nested under `_OPENACC` in the `CMakelists` file.